### PR TITLE
Goulden datatrends

### PR DESCRIPTION
**Pull request by IJBG**
This pull request includes: 

- my work on generating species plots with ggplot from last semester (in the scratch folder) - is it alright in the scratch folder or should I move it somewhere else? 
- ebird taxonomy 2021 download link broke, rather than inserting a new link created an inst/taxonomy folder and downloaded v2022. If a new version is downloaded later, the import_ebird_taxonomy function will pick the latest year version. 
- updated README with instructions for updating the files every year 
- fixed common and scientific name errors (ie: (domestic type), [Cardinal group], and subspecies like pandion haliaetus carolinensis) in the mbbs_county rdas

### CODE REVIEW

**Comment by bsaul** [comment]
> * is it alright in the scratch folder or should I move it somewhere else?
> 

Let's ignore scratch files by adding an extry to `.gitignore`; e.g. something like

```
**/scratch/*
```
or 

```
scratch**
```

File: README.md [hunk 1]
```diff
@@ -9,3 +9,57 @@ This repository contains code to download and analyze data from the [Mini-Bird B
 ## Notes on the analysis data files
 
 * Prio
```
Comment (bsaul) [inline README.md:11]: This line can be deleted. Looks like someone (probably me) started to write something but didn't finish it.

File: README.md [hunk 2]
```diff
@@ -9,3 +9,57 @@ This repository contains code to download and analyze data from the [Mini-Bird B
...
+
+
+### Instructions for updating the data
```
Comment (bsaul) [inline README.md:14]: General comment on this file - make sure that the markdown formats correctly. For example, you can see how GitHub flavored markdown renders the file by clicking `view file`. Here's the rendering of the latest commit: 
https://github.com/nc-minibbs/mbbs/blob/33b11e198827b5112a1b2d44bedd2e5c640f0b57/README.md.

I would put each step in a section header, as in 

```md
#### Step 1: Checking and QC'ing ebird data
```

File: README.md [hunk 3]
```diff
@@ -9,3 +9,57 @@ This repository contains code to download and analyze data from the [Mini-Bird B
...
+After a fresh new year and a new series of surveys, here's the walkthrough of how to update this repository with the latest information
+
+STEP1: Checking and QC'ing the ebird data
```
Comment (bsaul) [inline README.md:17]: I would put each step in a section header, as in 

```md
#### Step 1: Checking and QC'ing ebird data
```

File: DESCRIPTION [hunk 4]
```diff
@@ -1,6 +1,6 @@
 Package: mbbs
 Title: Data and data processing functions for the Mini-Bird Breeding Survery
-Version: 0.3.0.0
+Version: 2022.0
 Authors@R: c(
```
Comment (bsaul) [inline DESCRIPTION:4]: @IJBG be sure to add yourself to the author list!

File: DESCRIPTION [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Package: mbbs
 Title: Data and data processing functions for the Mini-Bird Breeding Survery
-Version: 0.3.0.0
+Version: 2022.0
```
Comment (bsaul) [inline DESCRIPTION:3]: Just noting that if this PR is merged before #31 then we'll need to change the version number in #31. If #31 is merged first, then we're fine.

**Comment by bsaul** [comment]
@IJBG - Looks great. I just reviewed and made a few comments. I didn't read the data processing instructions very thoroughly -- if you'd like me to give a closer read, I'm happy to.

**Comment by IJBG** [comment]
I've fixed the formatting for the README and added myself to the author list :) I think it's ready to merge after #31

**Comment by bsaul** [comment]
> I've fixed the formatting for the README and added myself to the author list :) I think it's ready to merge after #31

Great!

**Review by bsaul — APPROVED** [review]

File: DESCRIPTION (hunk 5 above)
Comment (bsaul) [reply DESCRIPTION:3]: Fixed in c97579dcb406611d51e59ec8f87cac815df3d2c2

**Comment by bsaul** [comment]
@IJBG - FYI I removed the `inst/scratch/*` files from the git database using `git rm --cached inst/scratch/*`. These files *should* remain in your local directory after this change.

**Comment by bsaul** [comment]
Also @IJBG thank you for the PR!

**Pull request merged**